premis_event_service/config/settings/base.py

```diff
@@ -0,0 +1,88 @@
+'''
+Default values for various app-defined settings
+
+Formula: FOO = getattr(settings, 'FOO', "default_value")
+
+These settings values will be used only if they are not found in the project
+settings.py module. If you want to override any of these values, just redefine
+them in that file rather than here.
+'''
+INSTALLED_APPS = (
```

Comment:
I don't think we should have all of the Django project level settings here in the premis_event_service app. While we need settings for a Django project for testing purposes, we probably don't need them in here. I think it makes sense to retain the functionality described above regarding attempting to get settings at a project level settings.py and using the ones in the app only if they are not otherwise set.